src/Service/DataSynchronizationService.php

```diff
@@ -73,7 +73,7 @@ public function syncProjects(callable $progressCallback, DataProvider $dataProvi
 
             foreach ($projectDatum->versions as $versionData) {
                 foreach ($versionData as $versionDatum) {
-                    $version = $this->versionRepository->findOneBy(['projectTrackerId' => $versionDatum->projectTrackerId, 'dataProvider' => $dataProvider]);
```

Comment:
But VersionData does not have an id property?
https://github.com/itk-dev/economics/blob/develop/src/Model/Invoices/VersionData.php#L12

Reply:
No, but `$versionDatum` (which is an instance of `SprintReportVersion`) does.